spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -90,6 +93,7 @@ public async Task InitializeAsync()
     public Task DisposeAsync()
     {
         DeleteInstance(InstanceIdWithProcessingUnits);
+        DeleteInstance(InstanceIdWithMultiRegion);
```

Comment:
That would from a technical viewpoint certainly be possible, but I don't know what the possible resource limits are for the test project. @amanda-tarafa and @skuruppu Do you know if there are any limits, and if so, what they are? (I know that we keep the default test instance alive, and that it does not cause any problems with any limits, but removing the deletion of both of these would triple the number of instances that is kept alive)

Reply:
I think it can hit the max limit easily if we don't delete the instances. Also, we had an issue to see a large amount of empty LCI instances before. I would not recommend to keep them alive.

Reply:
They'll be deleted on the next test run. And we can be more aggresive deleting old ones (we currently delete the 10 oldest I think). Although I prefer my suggestion of keeping them "for a while", as it is sometimes useful to inspect the resource used by a failed test, I'm not terrible bothered by deleting them here.